enterprise/metrics/src/main/java/org/neo4j/metrics/source/JvmMetrics.java

```diff
@@ -37,10 +37,10 @@
 
 import static com.codahale.metrics.MetricRegistry.name;
 
-@Documented( ".Java Virtual Machine Metrics ++\n" +
-             "These are environment dependent and they may vary on different hardware and with JVM configurations.\n" +
+@Documented( "=== Java Virtual Machine Metrics\n\n" +
```

Comment:
@nawroth Curiosity question: why is this title different with respect the other metrics?